src/Blueprint/Blueprint/Models/PSBlueprintTargetScope.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Blueprint.Models
+{
+    public enum PSBlueprintTargetScope
+    {
+        Unknown = 0,
+        Subscription = 1,
+        ManagementGroup = 2,
```

Comment:
>ManagementGroup = 2, [](start = 8, length = 20)

This isn't allowed... yet.

Reply:
Removing.. till then :)

---
In reply to: [257122123](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257122123) [](ancestors = 257122123)